test/torch/model/test_jit.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+import pytest
+
+import torch
+
+from gluonts.torch.model.deepar import DeepARModel
+from gluonts.torch.model.tft import TemporalFusionTransformerModel
+
+
+def get_model_and_input(model):
+    return model, tuple(
+        [
+            torch.zeros(shape, dtype=model.input_types()[name])
+            for (name, shape) in model.input_shapes().items()
```

Comment:
I think it would be nice to not have two different methods to get information on values.

Reply:
OK, I guess that can be addressed separately?

Reply:
Yes!